cmd/analyze/analyze.go

```diff
@@ -142,4 +144,6 @@ func init() {
 	AnalyzeCmd.Flags().BoolVarP(&customAnalysis, "custom-analysis", "z", false, "Enable custom analyzers")
 	// add custom headers flag
 	AnalyzeCmd.Flags().StringSliceVarP(&customHeaders, "custom-headers", "r", []string{}, "Custom Headers, <key>:<value> (e.g CustomHeaderKey:CustomHeaderValue AnotherHeader:AnotherValue)")
+	// label selector flag
+	AnalyzeCmd.Flags().StringVarP(&labelSelector, "selector", "L", "", "Label selector (label query) to filter on, supports '=', '==', and '!='. (e.g. -l key1=value1,key2=value2). Matching objects must satisfy all of the specified label constraints.")
```

Comment:
I think accepting a slice of label key pairs, will make the feature even more useful

Reply:
oh so you already support multiple key value pairs for labels !

awesome, then my only suggestion is to add a test case with more than one label just to sanity check it 👍

Reply:
> What will it be like?
I was just  thinking that we can work with `StringSliceVarP` and have it as 
`./bin/k8sgpt analyze -L app.kubernetes.io/part-of=argocd app.kubernetes.io/name=argocd-application-controller` 

but if you already support multiple labels, I think is fine as it is.